src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import java.util.function.BinaryOperator;
+
+public class StringCalculator {
+
+    private int result;
+    private String[] splitExpression;
+
+    public int calculate(String expression) {
+        validateInput(expression);
+
+        splitExpression = expression.split(" ");
+        result = Integer.parseInt(splitExpression[0]);
+
+        for (int idx = 1; idx < splitExpression.length; idx += 2) {
+            calculateOneByOne(idx);
+        }
+
+        return result;
+    }
+
+    private void calculateOneByOne(int idx) {
+        BinaryOperator<Integer> calculateFunction = distinguishOperator(splitExpression[idx]);
+        int operand = Integer.parseInt(splitExpression[idx + 1]);
+
+        result = calculateFunction.apply(result, operand);
+    }
+
+    private void validateInput(String expression) {
+        if (expression == null || expression.isEmpty())
+            throw new IllegalArgumentException("입력은 공백이면 안됩니다.");
+    }
```

Comment:
```suggestion
        if (expression == null || expression.isEmpty()) {
                throw new IllegalArgumentException("입력은 공백이면 안됩니다.");
        }
    }
```

자바에서는 하나의 괄호도 작성해주는 것이 컨벤션입니다 :)